### PR TITLE
[Backport] Re-enable XML as request and response types within the SwaggerUI

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Original Pull Request 
 https://github.com/magento/magento2/pull/23025

This MR adds the `application/xml` option to the SwaggerUI for requests and responses.

### Description (*)

The Magento WebApi is capable of handling XML as exchange format beside JSON but this format option is not shown/available in the SwaggerUI for requests and responses (only `application/json` is shown).

### Manual testing scenarios (*)

1. Open the SwaggerUI in the browser `/swagger`
2. Select `application/xml` as format for the request parameters AND/OR
3. Select `application/xml` as format for the response data

Screenshot 2019-05-27 at 17 08 05

Screenshot 2019-05-27 at 17 08 13

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-23190.instances.magento-community.engineering).
Admin access: [https://pr-23190.instances.magento-community.engineering/admin](https://pr-23190.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-5265](https://jira.corp.magento.com/browse/ENGCOM-5265) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

File: app/code/Magento/Webapi/Model/Rest/Swagger/Generator.php [hunk 1]
```diff
@@ -41,6 +41,11 @@ class Generator extends AbstractSchemaGenerator
     /** Array signifier */
     const ARRAY_SIGNIFIER = '[0]';
 
+    /**
+     * Wrapper node for XML requests
+     */
+    private const XML_SCHEMA_PARAMWRAPPER = 'request';
```
Comment (Nazar65) [inline app/code/Magento/Webapi/Model/Rest/Swagger/Generator.php]: Hi @speedy008  Only PHP > 7.1: private const

**Review by Nazar65 — CHANGES_REQUESTED** [review]

File: app/code/Magento/Webapi/Model/Rest/Swagger/Generator.php [hunk 2]
```diff
@@ -41,6 +41,11 @@ class Generator extends AbstractSchemaGenerator
     /** Array signifier */
     const ARRAY_SIGNIFIER = '[0]';
 
+    /**
+     * Wrapper node for XML requests
+     */
+    private XML_SCHEMA_PARAMWRAPPER = 'request';
```
Comment (sidolov) [inline app/code/Magento/Webapi/Model/Rest/Swagger/Generator.php]: Syntax is incorrect

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by VladimirZaets** [comment]
Hi @speedy008. The branch `2.2-develop` will be closed for contribution from **July 15, 2019**. Please finish all your Pull Requests till this date, otherwise, they will be closed.
Thanks for collaboration.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-5265](https://jira.corp.magento.com/browse/ENGCOM-5265) has been created to process this Pull Request
:eight_spoked_asterisk: @sivaschenko, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**